Sources/LiveViewNative/BuiltinRegistry.swift

```diff
@@ -14,43 +14,43 @@ struct BuiltinRegistry {
     @ViewBuilder
     static func lookup<R: CustomRegistry>(_ name: String, _ element: ElementNode, context: LiveContext<R>) -> some View {
         switch name {
-        case "textfield":
+        case "text-field":
             TextField<R>(element: element, context: context)
-        case "securefield":
+        case "secure-field":
             SecureField<R>(element: element, context: context)
         case "text":
             Text(context: context)
-        case "hstack":
+        case "h-stack":
```

Comment:
I do wonder if we should alias this to `hstack` as well as the other `-stack` views. I'm all for consistency but in the case of these specific views it does feel better writing:

```html
<hstack>
...
</hstack>
```

rather than

```html
<h-stack>
...
</h-stack>
```

Thoughts? So the idea would be to preserver the `h-stack` but offer the nicer alternative

Reply:
For the stacks in particular, I'm fine permitting both options. `hstack` does read nicer than `h-stack`, and stacks are very common

Reply:
yeah I don't think this should be allowed beyond stacks

Reply:
I enabled this for stacks.